### PR TITLE
Feature 3

### PR DESCRIPTION
**Pull request by Namrata-J**
#4 here i have added functionalities, like edit, save, add tags, background color, filter, delete & restore, and listing them in different sections based on their category.
for example, labeled will be shown in labeled section, archived would be shown in archived section, deleted would be shown in trash section.

Please review it, thankyou!!

Live preview: https://tknotes.netlify.app/

### CODE REVIEW

File: src/components/FilterAndSortPopUp/FilterAndSortPopUp.jsx [hunk 1]
```diff
@@ -1,50 +1,65 @@
...
+                    </select>
+
+                    {
+                        labeledNotes.length > 0 ?
+                            <>
+                                <div className="tn_popUp-subheading" >Select Labels</div>
+                                {
+                                    uniqueLabels.map((label, index) => {
+                                        return (
+                                            <label className="tn_select-label" htmlFor={label} key={index}>
```
Comment (JantuDeb) [inline src/components/FilterAndSortPopUp/FilterAndSortPopUp.jsx:50]: avoid using index for` key`

**Review by JantuDeb — APPROVED** [review]
Overall this looks good

**Review by ultronjs — APPROVED** [review]
Good to Merge.
Code is clean 
The contexts are well structured
Suggestion:-
Can export icons from one common file like icons.js to reduce boilerplate code in every file and same can be done for context.

File: src/components/FilterAndSortPopUp/FilterAndSortPopUp.jsx (hunk 1 above)
Comment (Namrata-J) [reply src/components/FilterAndSortPopUp/FilterAndSortPopUp.jsx:50]: okk will change it!! and will use .id instead!!

**Comment by Namrata-J** [comment]
> Good to Merge. Code is clean The contexts are well structured Suggestion:- Can export icons from one common file like icons.js to reduce boilerplate code in every file and same can be done for context.

okkay will do the refactoring!!

**Pull request merged**